profiles/arch/amd64/package.use.mask

```diff
@@ -18,6 +18,10 @@
 
 #--- END OF EXAMPLES ---
 
+# Karol Herbst <gentoo@karolherbst.de>
```

Comment:
Ok, we figured out that my name will be in ChangeLogs, so it should be here as well. So please change that to:

    # Michał Górny <mgorny@gentoo.org> (01 Mar 2015)

and I shall be able to proceed :).